### PR TITLE
add hocon

### PR DESCRIPTION
**Pull request by antosha417**
add support for hocon files

### CODE REVIEW

**Review by theHamsta — APPROVED** [review]
Seems to work really nice! Thank you!

If you want you can also add example files with highlight annotation to `tests/queries/highlights` but it's not necessary. Just in case you already experimented with some example files to see whether it's working.

**Comment by antosha417** [comment]
@theHamsta, I added highlights tests

**Comment by antosha417** [comment]
I think I should add .conf to ftdetect for tests to pass?

**Comment by theHamsta** [comment]
Can you add a ftdetect to `scripts/minimal_init.lua` https://github.com/theHamsta/nvim-treesitter/blob/dd2524171029169c410c92b028f610fa67ce2beb/scripts/minimal_init.lua#L4 ?

File: ftdetect/hocon.vim [hunk 1]
```diff
@@ -0,0 +1 @@
+au BufRead,BufNewFile *.conf set filetype=hocon
```
Comment (theHamsta) [inline ftdetect/hocon.vim:1]: please don't add this on such a generic file ending. For the test to pass, just add it to scripts/minimal_init.lua

**Review by theHamsta — CHANGES_REQUESTED** [review]
please don't add this on such a generic file ending. For the test to pass, just add it to scripts/minimal_init.lua

File: ftdetect/hocon.vim (hunk 1 above)
Comment (antosha417) [reply ftdetect/hocon.vim:1]: Ok. Will do tomorrow. Thanks

Comment (antosha417) [reply ftdetect/hocon.vim]: done. Now tests should pass

**Review by theHamsta — APPROVED** [review]

**Pull request merged**